### PR TITLE
[19.9] Add missing test annotations

### PR DESCRIPTION
**Pull request by lboynton**
These tests weren't being run!

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Please could you send this to 19.9. :)

File: tests/Api/ProjectsTest.php [hunk 1]
```diff
@@ -1957,6 +1960,9 @@ public function shoudGetApprovalsConfiguration(): void
         $this->assertEquals($expectedArray, $api->approvalsConfiguration(1));
     }
 
+    /**
+     * @test
+     */
     public function shoudGetApprovalRules(): void
```
Comment (GrahamCampbell) [inline tests/Api/ProjectsTest.php]: `shoudGetApprovalsRules`

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: tests/Api/ProjectsTest.php (hunk 1 above)
Comment (lboynton) [reply tests/Api/ProjectsTest.php]: Done :)

**Comment by lboynton** [comment]
> Please could you send this to 19.9. :)

Rebased on 9.19, hope that's correct :)

**Comment by GrahamCampbell** [comment]
Yep, this looks great. :+1:

**Comment by glensc** [comment]
I find it more convenient to use matching pattern for method names: `testSomething`, so they get picked up automatically. less clutter as well.

**Pull request merged**